src/Protocol/Authenticator.cpp

```diff
@@ -223,7 +223,7 @@ bool cAuthenticator::GetPlayerProperties(const AString & a_UUID, Json::Value & a
 	AString Request;
 	Request += "GET " + ActualAddress + " HTTP/1.0\r\n";
 	Request += "Host: " + m_Server + "\r\n";
-	Request += "User-Agent: MCServer\r\n";
+	Request += "User-Agent: Cuberite\r\n";
```

Comment:
General convention is that when changing name you keep the old name in you're user agent so that anything that matched the old useragent continues to match.